test/testcafe/enLeaks/EnglishLeaks_spec.js

```diff
@@ -0,0 +1,164 @@
+  'safe-mode-optional-enrollment.json',
+  'safe-mode-credential-enrollment-intent.json',
+  'oda-enrollment-ios.json',
+  'oda-enrollment-android.json',
+  'identify-with-third-party-idps.json',
+  'identify-with-only-one-third-party-idp-app-user.json',
+  'identify-with-only-one-third-party-idp.json',
+  'identify-with-no-sso-extension.json',
+  'identify-with-device-probing-loopback.json',
+  'identify-with-device-probing-loopback-3.json',
+  'identify-with-device-probing-loopback-2.json',
+  'identify-with-device-launch-authenticator.json',
+  'identify-with-apple-redirect-sso-extension.json',
+  'identify-unknown-user.json',
+  'error-user-is-not-assigned.json',
+  'error-unlock-account.json',
+  'error-session-expired.json',
+  'error-safe-mode-polling.json',
+  'error-pre-versioning-ff-session-expired.json',
+  'error-okta-verify-totp.json',
+  'error-internal-server-error.json',
+  'error-identify-access-denied.json',
+  'error-google-authenticator-otp.json',
+  'error-forgot-password.json',
+  'error-email-verify.json',
+  'error-authenticator-verify-password.json',
+  'error-authenticator-verification-custom-otp.json',
+  'error-authenticator-enroll-custom-otp.json',
+  'error-403-security-access-denied.json',
+  'consent-enduser.json',
+  'consent-admin.json',
+  'authenticator-verification-symantec-vip.json',
+  'authenticator-verification-select-authenticator.json',
+  'authenticator-verification-phone-voice-no-profile.json',
+  'authenticator-verification-phone-sms-no-profile.json',
+  'authenticator-verification-okta-verify-signed-nonce-loopback.json',
+  'authenticator-verification-okta-verify-signed-nonce-custom-uri.json',
+  'authenticator-verification-okta-verify-reject-push.json',
+  'authenticator-verification-number-challenge.json',
+  'authenticator-verification-data-phone-voice-then-sms.json',
+  'authenticator-verification-data-phone-voice-only.json',
+  'authenticator-verification-data-phone-sms-then-voice.json',
+  'authenticator-reset-password.json',
+  'authenticator-expiry-warning-password.json',
+  'authenticator-expired-password.json',
+  'authenticator-expired-password-with-enrollment-authenticator.json',
+  'authenticator-expired-password-no-complexity.json',
+  'authenticator-enroll-webauthn.json',
+  'authenticator-enroll-symantec-vip.json',
+  'authenticator-enroll-select-authenticator.json',
+  'authenticator-enroll-phone.json',
+  'authenticator-enroll-phone-voice.json',
+  'authenticator-enroll-ov-via-sms.json',
+  'authenticator-enroll-ov-via-email.json',
+  'authenticator-enroll-google-authenticator.json',
+  'authenticator-enroll-email.json',
+  'authenticator-enroll-data-phone.json',
+  'authenticator-enroll-data-phone-voice.json',
+  'terminal-enduser-email-consent-denied.json',
+  'terminal-return-email-consent-denied.json',
+  'terminal-return-email-consent.json',
+  'email-challenge-consent.json',
+  'error-authenticator-enroll-idp.json',
+  'error-authenticator-verification-idp.json',
```

Comment:
future: We should make sure these files exist via a lint rule or attempt to open them in our processing logic. We should try to avoid these files being removed or renamed.

Reply:
Yeah, I am aware of that have captured it in my [wiki](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/1940983617/SIW+-+Localization+OIE#How-to-ignore-tests-using-test-metadata-%3F). If the file gets renamed then the mock will get included in the test suite and the test will fail.